bom/runtime/pom.xml

```diff
@@ -2292,6 +2292,20 @@
                 <groupId>com.amazonaws.serverless</groupId>
                 <artifactId>aws-serverless-java-container-core</artifactId>
                 <version>${aws-lambda-serverless-java-container.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>commons-logging</groupId>
```

Comment:
Isn't this artifact banned?  We'd want (iirc) `org.jboss.logging:commons-logging-jboss-logging` (or maybe `-logmanager`, I forget which one we're using`.

Reply:
without the exclusion I get enforcer errors.

Reply:
Oh! My bad, I'm reading this on the go and didn't notice that it's an exclusion, not a dependency.  Carry on!